test/node/lib/test_aliases.js

```diff
@@ -1,211 +1,161 @@
 /* global describe, it */
 
-var assert  = require('proclaim');
-var AliasResolver = require('../../../lib/aliases');
+const assert = require('proclaim');
+const setsToArrays = require('../utils/sets_to_arrays');
```

Comment:
It seems it doesn't -- https://github.com/Financial-Times/polyfill-service/pull/903#discussion_r84587739